docs/modules/device-lab-management/pages/deviceConnect/retrieve-deviceconnect-logs.adoc

```diff
@@ -40,8 +40,6 @@ Log into the deviceConnect portal, then click the *Systems* tab.
 
 Under System logs, choose the duration to gather the logs (the default is 2 days), then click *Save logs*.
 
-image:device-lab-management:dc-logs-save-logs-button-gigafox.PNG[]
-
```

Comment:
@madison-oates it seems you removed the reference to this picture instead of adding alt text?

Reply:
Updated image